scripts/validations.py

```diff
@@ -0,0 +1,128 @@
+import itertools
+import json
+import sys
+
+ACTION_CONTEXT = 'context'
+ACTION_COUNT = 'count'
+ACTIONS = (ACTION_CONTEXT, ACTION_COUNT)
+
+UNLISTED_ADDONS = set()
+
+
+def parse_validations(results):
+    return (json.loads(result) for result in results)
+
+
+def unlisted_validations(results, unlisted_addons=None):
+    if unlisted_addons is None:
+        unlisted_addons = UNLISTED_ADDONS
+    return (result
+            for result in results
+            if (not result['metadata'].get('listed', True)
+                or result['metadata'].get('id') in unlisted_addons))
+
+
+def severe_validations(results):
+    return (result
+            for result in results
+            if (result['signing_summary']['high'] > 0 or
+                result['signing_summary']['medium'] > 0 or
+                result['signing_summary']['low'] > 0))
+
+
+def error_messages(results):
+    return ({'addon': result['metadata'].get('id', 'unknown-addon'),
```

Comment:
I tried to find a common error between these addons and they didn't all have https://github.com/mozilla/amo-validator/blob/186e68402f2c95bd433284e37c3d4ca5c0f45eec/validator/testcases/installrdf.py#L193 or https://github.com/mozilla/amo-validator/blob/186e68402f2c95bd433284e37c3d4ca5c0f45eec/validator/metadata_helpers.py#L43 which is what I was expecting.